### PR TITLE
fix: border style name and lang selector design

### PR DESCRIPTION
**Pull request by tlebel**
### PR check list

- [x] The title of the PR is formatted following [conventional commits](https://www.conventionalcommits.org/) format 
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Functionality has been verified and is ready to be code reviewed

### Relevant issues

N/A

## What is the current behavior?

- Current lang is not highlighted
- When we click on a lang the border becomes blue 
- The border names around the preview cards is wrong 

image

## What is the new behavior?

Fixed the above mentioned items and simplified the language selector: 

image

## Other information

**Does this PR introduce a breaking change?** 

No

### CODE REVIEW

File: components/event-preview/event-preview.vue [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <article
     :class="
       isIncoming
-        ? 'px-8 py-2 mb-10 border-solid border-8 border-green-500'
-        : 'px-8 py-2 mb-10 border-solid border-4 grey-border-200'
+        ? 'px-8 py-2 mb-10 border-solid border-8 border-mtl-green-500'
+        : 'px-8 py-2 mb-10 border-solid border-4 border-grey-border-200'
```
Comment (fuentesloic) [inline components/event-preview/event-preview.vue:6]: Should we not have a `border-mtl-grey-200` instead of `border-grey-border-200`?

Comment (tlebel) [reply components/event-preview/event-preview.vue]: Very good point! I also found more "gray" color issues while fixing this. Thanks!

**Comment by tlebel** [comment]
> @tlebel as we are not using the flag images anymore, can we delete them from the assets folder ? otherwise, looks good to me

Very good point!! Will do! Thanks!

**Comment by tlebel** [comment]
@aurelienroux  @fuentesloic : I'd need your approval again since pushing a commit cancel your previous approval 😣 

I remove that restriction... 
I trust we will behave professionally enough not to bug for re-approvals everytime haha!

**Pull request merged**